README.md

```diff
@@ -13,11 +13,11 @@ This directory contains the following helm charts.
 
 ## Prerequisites
 
-* Helm 3.5+
+You need the [`helm`](https://helm.sh/docs/intro/install/) command. See [Package manager section in Requirements document](https://scalardb.scalar-labs.com/docs/latest/requirements/#package-manager) for details on supported Helm versions.
```

Comment:
I think we'll be adding a Requirements doc to the ScalarDL docs. Should we include a "Package manager" section in that doc so that we can link to it here?

Reply:
Ah, yes. I think we should create a `Requirements` document of ScalarDL that includes a `Package manager` section. And, we should link to the document from this README.